demo-vue/src/validators/formValidator.js

```diff
@@ -0,0 +1,19 @@
+import { form, min, required, compare, every } from 'strickland';
+import { formDefinition } from '../../../demo/src/formValidator';
+
+const formDefinitionExtended = {
+  ...formDefinition,
+  age: [
+    required({ message: 'Required' }),
+    min({ min: 18, message: 'You must be 18+.' })
+  ],
+  gender: required({ message: 'Required' }),
+  acceptsTerms: [
+    every(
+      [required(), compare(true)],
+      { message: 'You must accept the terms and conditions.' }
+    )
+  ]
```

Comment:
A couple of notes here:

1. You don't need to wrap `every` in another array; you could just have `acceptTerms: every([required(), compare(true)], { message: 'You must accept the terms and conditions.' })`
2. But `required` will indeed take care of this on its own, and since we call this specific scenario out in the docs for `required`, I'd like to illustrate that here with `acceptTerms: required({ message: 'You must accept the terms and conditions.' })`

Reply:
Done.